### PR TITLE
Update troubleshooting-dataflow-issues-connection-to-the-data-source.md

### PR DESCRIPTION
**Pull request by JoelSteph**
From MS CSS team: There was known issue with dataflows with cloud data sources whose DSNs have been modified. Dataflow by design doesn't delete the old connection but keeps the old and new connection which leads to slow Preview data load and unable to save changes issue. Recommending add a note at the end of doc for cloud data source based dataflow DSN changes

### CODE REVIEW

**Comment by JoelSteph** [comment]
@bensack : Creating PR for addition of known limitation and workaround based on CRI and PG recommendation. Please review and do reach out to me for queries

File: powerquery-docs/dataflows/troubleshooting-dataflow-issues-connection-to-the-data-source.md [hunk 1]
```diff
@@ -89,6 +89,11 @@ In order to trim the data sources, you'll need to take the following steps:
...
    ![Gateway selector.](media/troubleshooting-dataflow-issues-connection-to-the-data-source/gateway-selection.png)
 
+
 1. After you apply the change by selecting **OK**, repeat steps 1 through 4 to select the original gateway again.
 
 These steps essentially delete all the data source bindings for the dataflow. After finishing these steps, you might be asked to set up credentials. When you've set up the gateway and your credentials successfully, you effectively "trimmed" the data source bindings for the dataflow to just the ones that the dataflow is actually using.
+
+
+Note: 
+For dataflows with cloud data sources without on-premises data gateway, when the datasource name changes, there would be an old data source connection still existing which can be removed by going to Manage Connections and Gateways page > Connections > Remove the old/unused data source connection and then go to the Dataflow > Edit Tables and then click Save & Close for the Dataflows to be updated to current data source connection and remove older connection
```
Comment (Court72) [inline powerquery-docs/dataflows/troubleshooting-dataflow-issues-connection-to-the-data-source.md:99]:
```suggestion
> [!NOTE] 
> For dataflows with cloud data sources without on-premises data gateway, when the data source name changes, an old data source connection still exists. You can remove this connection by going to the Manage Connections and Gateways page > Connections > Remove the old/unused data source connection. Then go to the Dataflow > Edit Tables and select **Save & Close** for the Dataflows to be updated to the current data source connection and to remove the older connection. 

```

**Comment by Court72** [comment]
@bensack

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by bensack** [comment]
#sign-off

**Comment by Court72** [comment]
@bezhan-msft, @DougKlopfenstein 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

**Comment by DougKlopfenstein** [comment]
Hi @JoelSteph 
I made some changes to the note because the path to the options you describe was vague. Please check the additions I made for clarity to ensure that the note still contains the valid steps for removing the old connection. If everything is OK, just add a sign-off comment and the changes will be merged. Otherwise, you can edit the note so that it is correct.

**Review by JoelSteph — COMMENTED** [review]
For dataflows with cloud data sources without an on-premises data gateway, when the data source name changes, an old data source connection still exists. To remove this connection, in the Power BI service, select the settings icon, go to Manage Connections and Gateways > Connections, select the ellipsis menu by the connection name, then select Remove to remove the old/unused data source connection. Then go to the Home page, select the workspace, select the dataflow, and then select Edit from the ellipsis menu by the dataflow name. In the Table name page, select Edit tables. Then in the Power Query editor, select Save & Close for the dataflows to be updated to the current data source connection and to remove the older connection.


Suggest Change to:

For dataflows with cloud data sources without an on-premises data gateway, when the data source name changes, an old data source connection still exists. To remove this connection, in the Power BI service, select the settings icon, go to Manage Connections and Gateways > Connections, select the ellipsis menu by the connection name, then select Remove to remove the old/unused data source connection. Then go to the Home page, select the workspace, select the dataflow to open it. In the Table name page, select Edit tables. Then in the Power Query editor, select Save & Close for the dataflows to be updated to the current data source connection and to remove the older connection.

**Comment by JoelSteph** [comment]
Thanks @DougKlopfenstein , I did quick test and found that we can make one small change. On the fourth line, we do not need to click Edit from the ellipsis menu by the dataflow name. We need to just click on Dataflow to open it and then in the Power Query editor click Save and close to refresh. Once that is changed, i think we should be good :)

**Comment by DougKlopfenstein** [comment]
#sign-off

**Comment by DougKlopfenstein** [comment]
Hi @JoelSteph 
Looks like you will have to add a sign off comment for this pull request to be merged.

**Comment by JoelSteph** [comment]
#sign-off

**Pull request merged**